sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -247,6 +247,20 @@
     </resources>
 
     <plugins>
+      <!-- This plugin checks dependencies for android API level compatibility -->
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>animal-sniffer-maven-plugin</artifactId>
+        <version>1.23</version> <!-- {x-version-update;org.codehaus.mojo:animal-sniffer-maven-plugin;external_dependency} -->
+        <configuration>
```

Comment:
Consider wrapping this configuration in <executions> section and run it in `package` phase. Also, configure `failOnError` to `false`.

```xml
  <plugin>
    ...
    <executions>
      <execution>
        <phase>package</phase>
        <goals>
          <goal>check</goal>
        </goals>
        <configuration>
          <signature>
            ...
          </signature>
          <failOnError>false</failOnError>
        </configuration>
      </execution>
    </executions>
  </plugin>
```